deployment/build.centos.all

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+#= CentOS 7 all .rpm
+
+set -o errexit
+set -o xtrace
+
+# Move to source directory
+pushd ${SOURCE_DIR}
+
+cp -a yarn.lock /tmp/yarn.lock
```

Comment:
this is a dirty hack, why don't we use `npm ci` which preserves lockfile? or there was some other option that leaves lockfile in place (and uses it to actually freeze the versions for reproducibility)

cc @dkanada @thornbill

Reply:
This can be dealt with later.